source/_components/media_player.samsungtv.markdown

```diff
@@ -25,6 +25,8 @@ To add a TV to your installation, add the following to your `configuration.yaml`
 media_player:
   - platform: samsungtv
     host: 192.168.0.10
+    port: 8001
+    mac: 5c:49:7d:9a:6f:5d
 ```
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.